### PR TITLE
WIP: Modify latex_mini.md

### PR DESCRIPTION
**Pull request by jpherdi**
Here is a first version of a verbose and full manual for the latex-mini installation.

### CODE REVIEW

**Comment by chrbeckm** [comment]
Maybe detail the way to remove the first, failed, bigger TeX Installation? In case, people try that first.

File: install/downloads/latex_packages.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+babel babel-english babel-french babel-german beamer biber biblatex blindtext bookmark booktabs caption carlisle circuitikz csquotes enumitem environ epstopdf-pkg eso-pic extsizes fancyvrb fira firamath firamath-otf float fontawesome5 fontspec infwarerr koma-script latex latex-bin latexmk listings lstaddons luacode lualatex-math luatex luatex85 luatexbase mathtools metalogo mhchem microtype mleftright mwe ncctools oberdiek pdflscape pgfopts placeins siunitx standalone tabu tabulary tcolorbox texdoc threeparttable tikz tikz-feynman trimspaces type1cm unicode-math varwidth xits xkeyval xpatch xstring
```
Comment (maxnoe) [inline install/downloads/latex_packages.txt:1]: can we make this one line per package?

File: install/latex_mini.md [hunk 2]
```diff
@@ -7,29 +7,38 @@ Wir empfehlen die komplette Version zu installieren, falls der vorhandene Speich
...
-- **Z.22** Wechseln in das Installationsverzeichnis
-- **Z.24** Installation der minimal benötigten Pakete
-- **Z.93** Setzen der update Optionen
+Eine Liste aller notwendiger Latex Packages, die wir brauchen werden, bekommt ihr mit
+```
+curl -fLO https://toolbox.pep-dortmund.org/install/downloads/latex_packages.txt
+```
+Die Installation kann dann durchgeführt werden mit 
+```
+<latex_packages.txt | xargs tlmgr install
```
Comment (maxnoe) [inline install/latex_mini.md:38]: `xargs < latex_packages.txt`

**Comment by jpherdi** [comment]
Btw running this would always give an error with `tikz` in the list. The package is called pgf.

**Comment by jpherdi** [comment]
I changed the inline code, because the pink is very ugly. No I changed it in the way it is on e.g. stackoverflow.

**Review by chrbeckm — APPROVED** [review]

**Pull request merged**